test/ipynb/mpl/test_circuit_matplotlib_drawer.py

```diff
@@ -114,7 +124,8 @@ def test_plot_barriers(self):
 
         # check for barriers
         circuit.h(q[0])
-        circuit.barrier()
+        circuit.barrier(0)
+        circuit.h(q[0])
```

Comment:
Why this change? If the new situation is particularly interesting, I would add a new test.

Reply:
Just thought since there was a recent fix to displaying partial barriers, it was good to have one. Could be split out to another test, course.

Reply:
split in 22383384cfaf1d5bc44b2241984636de997e7fe1